synapse/models/orgs.py

```diff
@@ -240,6 +251,38 @@ def getModelDefs(self):
                     }),
                 )),
                 ('ou:name', {}, ()),
+                ('ou:contract', {}, (
```

Comment:
Do we want to add :loc to capture where the contract took place? this would allow us to lift all RU contracts for example

Reply:
I'd prefer to delay any kind of geospatial/geopolitical jurisdiction of the contract model for now.  I think it'll be enough that it's potentially a contract between orgs that have a geopolitical association ( until we figure out more about the use case ).